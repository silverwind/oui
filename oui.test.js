import {execa} from "execa";

test("cli", async () => {
  let stdout, exitCode;

  ({stdout, exitCode} = await execa("node", ["./bin/oui.js", "20:37:06:12:34:56"]));
  expect(stdout).toMatchSnapshot();
  expect(exitCode).toEqual(0);

  ({stdout, exitCode} = await execa("node", ["./bin/oui.js", "20_37_06"]));
  expect(stdout).toMatchSnapshot();
  expect(exitCode).toEqual(0);

  ({stdout, exitCode} = await execa("node", ["./bin/oui.js", "203706"]));
  expect(stdout).toMatchSnapshot();
  expect(exitCode).toEqual(0);
});
