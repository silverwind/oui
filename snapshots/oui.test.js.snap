// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`cli 1`] = `
"Cisco Systems, Inc
80 West Tasman Drive
San Jose CA 94568
United States"
`;

exports[`cli 2`] = `
"Cisco Systems, Inc
80 West Tasman Drive
San Jose CA 94568
United States"
`;

exports[`cli 3`] = `
"Cisco Systems, Inc
80 West Tasman Drive
San Jose CA 94568
United States"
`;
